### PR TITLE
remove polka and add profile manager funcs

### PR DESCRIPTION
**Pull request by A-Harby**
### Description

- Polkadot extension files and pages were removed.
- New profile manager function were added.
- All the test cases were updated to use the new profile manager. 
- All new feature were added/updated.
- Docs were also update in every test case.

### CODE REVIEW

File: packages/dashboard/tests/frontend_selenium/tests/test_twin.py [hunk 1]
```diff
@@ -1,102 +1,69 @@
...
-    twin_page.accept_terms_conditions()
-    polka_page.authenticate_with_pass(password)
-    assert twin_page.wait_for('Accepted!')
-    assert twin_page.wait_for('Choose a Relay Address')
-
-
-def test_create_twin_relay(browser):
-    """
-      Test Case: TC932- check Balance
-      Test Case: TC1461 - Choose a Relay Address
```
Comment (mohamedamer453) [inline packages/dashboard/tests/frontend_selenium/tests/test_twin.py:42]: As mentioned in the `test_hompage` comment, twin creation should be covered.

**Review by mohamedamer453 — CHANGES_REQUESTED** [review]

File: packages/dashboard/tests/frontend_selenium/tests/conftest.py [hunk 2]
```diff
@@ -17,9 +17,8 @@ def browser():
 
     # Initialize the ChromeDriver instance with options
     options = webdriver.ChromeOptions()
-    options.add_extension('polka_extension_0_44_1_0.crx')  # PolkaDot Extension
-    driver = webdriver.Chrome(options=options, service=Service(ChromeDriverManager().install()))
```
[the remark this answers is not in the document]
Comment (A-Harby) [reply packages/dashboard/tests/frontend_selenium/tests/conftest.py:21]: Did you try to update packages? Selenium latest version is now handling `ChromeDriverManager` automatically.

File: packages/dashboard/tests/frontend_selenium/tests/test_homepage.py [hunk 3]
```diff
@@ -49,178 +30,121 @@ def test_manual_page(browser):
...
-    user = generate_string()
+    dashboard_page = before_test_setup(browser)
     password = generate_string()
-    polka_page.add_account(user, password)
-    assert user in dashboard_page.accounts_list()
+    dashboard_page.create_account()
+    dashboard_page.accept_terms_conditions()
+    dashboard_page.connect_your_wallet(password).click()
+    dashboard_page.logout_account()
+    assert dashboard_page.login_account(password).is_enabled() == True
```
[the remark this answers is not in the document]
Comment (A-Harby) [reply packages/dashboard/tests/frontend_selenium/tests/test_homepage.py:58]: 1. Selecting the relay is now being done by giving the twin the network relay, and later you will have the option is edit it from twin page.
2. I have added a test case for the twin details assertion from grid proxy but it's in the twin module.
3. Also edit twin relay has its own test case.

File: packages/dashboard/tests/frontend_selenium/tests/conftest.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (A-Harby) [reply packages/dashboard/tests/frontend_selenium/tests/conftest.py:21]: Based on workflow run https://github.com/threefoldtech/tfgrid-sdk-ts/actions/runs/5797487405/job/15713122420, It works fine will close this one but will also keep debugging for the reason or if it's show up again.

[the remark this answers is not in the document]
Comment (A-Harby) [reply packages/dashboard/tests/frontend_selenium/tests/conftest.py:21]: Based on workflow run https://github.com/threefoldtech/tfgrid-sdk-ts/actions/runs/5797487405/job/15713122420, It works fine will close this one but will also keep debugging for the reason or if it's show up again.

File: packages/dashboard/tests/frontend_selenium/tests/test_twin.py (hunk 1 above)
Comment (A-Harby) [reply packages/dashboard/tests/frontend_selenium/tests/test_twin.py:42]: https://github.com/threefoldtech/tfgrid-sdk-ts/pull/903#discussion_r1288618109

**Review by mohamedamer453 — APPROVED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**